pulumi/grapl/__main__.py

```diff
@@ -295,7 +295,7 @@ def main() -> None:
     )
 
     ConsulIntentions(
-        "grapl-core",
+        "consul-intentions",
```

Comment:
So actually since we're putting all of the intentions into one folder, this is fine. I don't know that setting these per folder are required since intentions can be created prior to the service actually existing